ydb/docs/en/core/troubleshooting/examples/schemas/overloaded-shard-1.md

```diff
@@ -0,0 +1,306 @@
+
+See that one of the data shards that processes queries for the `kv_test` table is loaded by 67%.
+
+Let's take a look at the `kv_test` table on the **Info** tab:
+
+![stock table info](./_assets/overloaded-shard-1/incident-ui-table-info.png)
+
+{% note warning %}
+
+The `kv_test` table was created with partitioning by load disabled and has only one partition.
+
+It means that only one data shard processes requests to this table. And we know that a data shard can process only one request at a time. This is really bad practice.
+
+{% endnote %}
+
+## Solution {#solution}
+
+We should enable partitioning by load for the `kv_test` table:
+
+1. In the Embedded UI, select the database.
+2. Open the **Query** tab.
+3. Run the following query:
+
+    ```sql
+    ALTER TABLE kv_test SET (
+        AUTO_PARTITIONING_BY_LOAD = ENABLED
+    );
+    ```
+
+## Aftermath {#aftermath}
+
+As soon as we enable automatic partitioning for the `kv_test` table, the overloaded data shard split in two.
+
+![shard distribution by load](./_assets/overloaded-shard-1/aftermath-grafana-shard-distribution-by-workload.png)
+
+{% cut "See the diagram description" %}
+
+The diagram shows that the number of data shards increased at about ##10:28:00##. Their workload does not exceed 40% judging by the bucket color.
+
+{% endcut %}
+
+![overloaded shard count](./_assets/overloaded-shard-1/aftermath-grafana-overloaded-shards.png)
+
+{% cut "See the diagram description" %}
+
+The overloaded shard disappeared from the diagram at about ##10:28:00##.
+
+{% endcut %}
+
+Two data shards are processing queries to the `kv_test` table now, none of them are overloaded:
+
+![overloaded shard count](./_assets/overloaded-shard-1/aftermath-ui-top-shards.png)
+
+Let's make sure the latencies are back to normal:
+
+![final latency percentiles](./_assets/overloaded-shard-1/aftermath-grafana-latency-percentiles.png)
+
+{% cut "See the diagram description" %}
+
+At about ##10:28:00## p50, p75, p95 latencies dropped almost to the original level. The p99 latency decrease is less pronounced, but still, it's a twofold drop.
+
+{% endcut %}
+
+![final latencies](./_assets/overloaded-shard-1/aftermath-grafana-latencies.png)
+
+{% cut "See the diagram description" %}
+
+The diagram shows that transactions are grouped into six buckets now. Approximately half of the transactions are back to `Bucket 1`, meaning that their latency is less than one millisecond. More than a third of the transactions are in `Bucket 2` with the latencies from one up to two milliseconds. One sixth of the transactions are in `Bucket 4`. The size of other buckets is insignificant.
+
+{% endcut %}
+
+The latencies are almost as low as they used to be before the workload increased. We did not add any additional hardware resources, just enabled automatic partitioning by load.
+
+#|
+|| Bucket name
+| Latencies, ms
+|
+Transactions,
+single overloaded data shard
+|
+Transactions,
+multiple data shards
```

Comment:
1. These headers look inconsistent as the first one with latencies is basically `"<label>, <unit of measurement>"`, while the rest are `"<unit of measurement>, <label>"`.
2. These are probably transactions **per second** instead of totals?